doc/spec-files.rst

```diff
@@ -343,9 +343,32 @@ For example modify the spec file this way::
           exclude_binaries=...
           )
 
+One-File mode
```

Comment:
- Should be something like "Converting a a spec-file from one-dir mode to one-file mode".
- Then there should be a sentence about why one would want to do this.
- A better place for this section would be behind the "Spec File Options for a Mac OS X Bundle"